### PR TITLE
Gathered data doc update - add some known previous locations

### PR DESCRIPTION
**Pull request by tremes**
Minor documentation update

## Categories

- [ ] Bugfix
- [ ] Enhancement
- [ ] Backporting
- [X] Others (CI, Infrastructure, Documentation)

## Sample Archive

No new data

## Documentation

- `docs/gathered-data.md `

## Unit Tests

No new tests

## Privacy

Yes. There are no sensitive data in the newly collected information.

## Breaking Changes

No

## References

https://issues.redhat.com/browse/???
https://bugzilla.redhat.com/show_bug.cgi?id=???
https://access.redhat.com/solutions/???

### CODE REVIEW

**Comment by tremes** [comment]
@jholecek-rh Can you please check? Is it sufficient to cover the https://issues.redhat.com/browse/CCXDEV-4976 pls?

File: docs/gathered-data.md [hunk 1]
```diff
@@ -159,6 +161,8 @@ collects all the ClusterOperators definitions and their resources.
 The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
 Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
 
+* Previous location of operators related resources in the older archives: config/clusteroperator/{kind}-{name}
```
Comment (tremes) [inline docs/gathered-data.md:164]: I am not sure if this was ever released, but it's probably true that we got some archives (with this structure) from CI/testing clusters.

**Comment by jholecek-rh** [comment]
> @jholecek-rh Can you please check? Is it sufficient to cover the https://issues.redhat.com/browse/CCXDEV-4976 pls?

The information is there. I would just suggest to arrange it a little different:
* List the current location first.
* Avoid the inaccurate "older archives". We are receiving very fresh archives from ancient Insights Operator versions. :slightly_smiling_face: 

How about this?
```
Location in archive: config/some/location
Location(s) in older versions: config/some/old/location, config/even/older/location
```

**Comment by tremes** [comment]
@jholecek-rh OK thanks for feedback!! Makes sense. I am going to update as suggested.

**Review by natiiix — APPROVED** [review]
/lgtm
Just a doc change, so no local testing was necessary.

**Comment by tremes** [comment]
Waiting for https://github.com/openshift/origin/pull/26264
/hold

**Pull request merged**